### ISSUE TITLE
Inform user of authentication errors inline

### ISSUE THREAD

**Issue by agoldingay** [hyperledger/sawtooth-next-directory#75]
[User Story] | Handling of Failed Logins
----------|----------
Epic Title | Advance Authentication #65 
User Story Title | Handling of Failed Logins
T-shirt size:  | S
User Story | "As a user of NexT, when I screw up my username, pw or have connectivity issues, I should be informed of WHY my login failed so that I can correct the issue and get back to work."
Description | Display an inline message under current view that explains in plain English the auth error a user may be encountered.
Acceptance Criteria/Tests | Demonstrate the informational information provided for all possible failure modes -- username not found, incorrect pw, server failed to respond, server responded with <reason>, etc... "  
Questions/Clarifications |

**Comment by PatrickLammers** [comment]
Pretty sure this is a dupe of #908.  Since that one seems to be getting fixed in S11, I'm moving this one along too.  Let's be sure the Acceptance Criteria for both are met
